ams/server/src/test/java/com/netease/arctic/server/optimizing/flow/view/MatchResult.java

```diff
@@ -60,7 +60,7 @@ public boolean isOk() {
 
   @Override
   public String toString() {
-    return Objects.toStringHelper(this)
+    return MoreObjects.toStringHelper(this)
```

Comment:
In the Arctic project, I suggest using the version of Guava Shaded by the Iceberg project as a dependency. Therefore, the imported classes here can be changed to: `org.apache.iceberg.relocated.com.google.common.base.MoreObjects`